src/JuliennedArrays.jl

```diff
@@ -69,17 +76,13 @@ end
 
 @inline slice_index(slices, indices) =
     setindex_unrolled(axes(slices.whole), indices, map(not, slices.alongs))
-@inline getindex(slices::Slices, indices::Int...) =
+@inline getindex(slices::Slices{T,N}, indices::Vararg{Int,N}) where {T,N} =
     view(slices.whole, slice_index(slices, indices)...)
-@inline setindex!(slices::Slices, value, indices::Int...) =
+@inline setindex!(slices::Slices{T,N}, value, indices::Vararg{Int,N}) where {T,N} =
     slices.whole[slice_index(slices, indices)...] = value
 
-@inline axis_or_1(switch, axis) =
-    if untyped(switch)
-        axis
-    else
-        1
-    end
+@inline axis_or_1(switch, axis) = untyped(switch) ? axis : 1
```

Comment:
I tend not to like to use the ternary operator. It's inconvenient if you want to add more lines to the if statement

Reply:
The idea here is to only use short-form function definitions when it can be fit into one line.

https://github.com/invenia/BlueStyle#method-definitions